source/GameInterface/Services/Villages/Messages/ChangeVillageState.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Common.Messaging;
+using ProtoBuf;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GameInterface.Services.Villages.Messages;
+
+
+/// <summary>
+/// This is used when the Client Sends the GameInterface data to change its state.
```

Comment:
We shouldn't reference server/client in the events as they are meant to be a generic API of sorts
```suggestion
/// Updates the village state
```